packages/autocomplete-js/src/__tests__/renderer.test.ts

```diff
@@ -1,3 +1,54 @@
+import { fireEvent, waitFor } from '@testing-library/dom';
+import {
+  createElement as preactCreateElement,
+  Fragment as preactFragment,
+} from 'preact';
+
+import { autocomplete } from '../autocomplete';
+
 describe('renderer', () => {
```

Comment:
I removed it because I thought it was redundant with other tests since it match a `provides default implementation` test, wdyt?

Reply:
Where is that test?

We can still write a test that is scoped to that file. We don't test `Fragment` here neither.

Reply:
I was referring to [this](https://github.com/algolia/autocomplete/pull/494/files#diff-624fd858985f046bc9057bd23e6df01f020a51b90f17736cfa4eba0e0261612dR15-R52) and [these](https://github.com/algolia/autocomplete/pull/494/files#diff-624fd858985f046bc9057bd23e6df01f020a51b90f17736cfa4eba0e0261612dR389-R453) tests but since they are scoped to the `render` method, I'll add one that combine both for the `renderer`.